lib/streamlit/web/bootstrap.py

```diff
@@ -228,35 +226,6 @@ def _print_url(is_running_hello: bool) -> None:
         cli_util.print_to_cli("")
 
 
-def _maybe_print_old_git_warning(main_script_path: str) -> None:
```

Comment:
This is a very old check. I don't think we need it to support this anymore for an 8-year-old git version.